zalando/zalando.go

```diff
@@ -122,6 +157,24 @@ func UidCheck(at []AccessTuple) func(tc *ginoauth2.TokenContainer, ctx *gin.Cont
 	}
 }
 
+// UidCheckNetHTTP is the net/http version of UidCheck
+func UidCheckNetHTTP(at []AccessTuple) ginoauth2.AccessCheckFunctionNetHTTP {
```

Comment:
Would it make sense to put this under a different package to not have the `NetHTTP` part in the function name?

Reply:
this would be a bigger refactoring, because of the dependencies, but yes you are right would be good to do